Search/Adapter/ZendLuceneAdapter.php

```diff
@@ -185,7 +185,9 @@ public function search(SearchQuery $searchQuery)
 
             $searcher->addIndex($this->getIndex($indexPath, false));
         }
+        if ($searchQuery->getSort() !== null) {
 
+        }
```

Comment:
It should not be hard to add sorting for zend search: http://framework.zend.com/manual/1.12/en/zend.search.lucene.searching.html#zend.search.lucene.searching.sorting